docs/input_formats/if_landxml.rst

```diff
@@ -0,0 +1,195 @@
+.. _LandXML website: http://www.landxml.org/
+
+.. _if_landxml:
+
+=========================
+:mod:`landxml` -- LandXML
+=========================
+
+.. moduleauthor:: Damien Gaignon
+.. versionadded:: 0.6
+
+LandXML is committed to providing an non-proprietary data standard (LandXML),
+driven by an consortium of partners for the inter-operability of data utilized
+within the Land Development industry.
+
+The official documentation about the format is provided on the `LandXML website`_.
+
+LandXML is a specialized XML (eXtensible Mark-up Language) data file format
+containing civil engineering and survey measurement data commonly used in the
+Land Development and Transportation Industries.
+
+For example::
+
+    <?xml version="1.0" encoding="utf-8"?>
+    <LandXML xmlns="http://www.landxml.org/schema/LandXML-1.1" time="15:49:09" date="2006-06-19" version="0" language="English" readOnly="false">
```

Comment:
the `xmlns` attribute should point to LandXML-1.2 as in the output format example

Reply:
Forgot this one ^^